### ISSUE TITLE
[Question] , data collection

### ISSUE THREAD

**Issue by yangyan010119** [PKU-Alignment/VLA-Arena#9]
### Required prerequisites

- [x] I have read the documentation <https://github.com/PKU-Alignment/VLA-Arena/tree/main/docs>.
- [x] I have searched the [Issue Tracker](https://github.com/PKU-Alignment/VLA-Arena/issues) and [Discussions](https://github.com/PKU-Alignment/VLA-Arena/discussions) that this hasn't already been reported. (+1 or comment there if it has.)
- [ ] Consider asking first in a [Discussion](https://github.com/PKU-Alignment/VLA-Arena/discussions/new).

### Questions

Dear author, hello! Your work has made a significant contribution to the community. I noticed that in your code, data collection needs to be done manually one by one. May I ask if you have any open-source datasets now? If the dataset can be open-sourced, it will be of great help to my work. I hope to communicate with you.

**Comment by yangyan010119** [comment]
> [您可以直接通过我们的拥抱面部组织访问VLA-Arena-S/M/L数据集:https://huggingface.co/VLA-Arena](https://huggingface.co/VLA-Arena)
> 
> 数据集分解:我们处理了分布在115个任务中的5750个轨迹(包括L0的60个任务和L1的55个任务)。基于此,我们提供三种变体:
> 
> VLA-Arena-S:10个轨迹/任务
> 
> VLA-Arena-M:30个轨迹/任务
> 
> VLA-Arena-L:50个轨迹/任务
作者您好，感谢您之前的回复，关于您在huggingface上提交的数据，我想用来做监督训练，但是对数据组成不是特别了解，我想将数据集划分成训练集和评估集，特此想咨询您，如何对数据集划分，是取整体排序的后百分之X即可作为评估集，还是每个任务的排序在一起的，需要按任务取某任务的后百分之X？还是说我需要用VLA-Arena-S:10个轨迹/任务作为评估集？

**Comment by learnerljh** [comment]
Hello, thank you for your question.

Regarding the dataset splitting, here are our suggestions:

Core Evaluation Standard: The primary goal of robotic tasks is to validate the effectiveness of the policy within the simulation scenes. Therefore, the gold standard for evaluation is to measure the actual success rate in the simulation, rather than relying solely on offline validation loss.

Regarding Offline Splitting: If you have specific needs for offline validation (e.g., monitoring training loss), you can split the dataset according to your own experimental requirements (for example, you can choose to randomly sample the last X% of each task as a validation set).

Hope this helps!

您好，感谢您的提问。

关于数据集的划分，我们建议您关注以下两点：

核心评估标准：机器人任务的核心在于在仿真场景中验证策略的实际执行效果。因此，最准确的评估方式是直接使用仿真环境测试模型的成功率，而非仅仅依赖离线验证集的 Loss。

关于离线划分：如果您有监控离线训练 Loss 等特殊需求，完全可以按照您自己的实验设计需要进行划分（例如，您可以选择按任务随机抽取后 X% 作为验证集）。

希望这对您有帮助！

**Comment by yangyan010119** [comment]
> Hello, thank you for your question.
> 
> Regarding the dataset splitting, here are our suggestions:
> 
> Core Evaluation Standard: The primary goal of robotic tasks is to validate the effectiveness of the policy within the simulation scenes. Therefore, the gold standard for evaluation is to measure the actual success rate in the simulation, rather than relying solely on offline validation loss.
> 
> Regarding Offline Splitting: If you have specific needs for offline validation (e.g., monitoring training loss), you can split the dataset according to your own experimental requirements (for example, you can choose to randomly sample the last X% of each task as a validation set).
> 
> Hope this helps!
> 
> 您好，感谢您的提问。
> 
> 关于数据集的划分，我们建议您关注以下两点：
> 
> 核心评估标准：机器人任务的核心在于在仿真场景中验证策略的实际执行效果。因此，最准确的评估方式是直接使用仿真环境测试模型的成功率，而非仅仅依赖离线验证集的 Loss。
> 
> 关于离线划分：如果您有监控离线训练 Loss 等特殊需求，完全可以按照您自己的实验设计需要进行划分（例如，您可以选择按任务随机抽取后 X% 作为验证集）。
> 
> 希望这对您有帮助！

Thank you very much for your reply. Your guidance has been of great help to me. I wish you all the best in your research!
